cmd/skaffold/app/flags/image.go

```diff
@@ -61,9 +56,18 @@ func (i *Images) Set(value string) error {
 	return nil
 }
 
-// Type Implements Type() method for pflag interface
-func (i *Images) Type() string {
-	return fmt.Sprintf("%T", i)
+// Replace Implements Replace() method for pflag SliceValue interface
+func (i *Images) Replace(images []string) error {
+	newImages := make([]image, len(images))
+	for _, value := range images {
+		a, err := convertImageToArtifact(value)
+		if err != nil {
+			return err
+		}
+		newImages = append(newImages, image{name:value, artifact:a})
+		i.images = newImages
+	}
+	return nil
 }
```

Comment:
```suggestion
		}
		newImages = append(newImages, image{name:value, artifact:a})
	}
	i.images = newImages
	return nil
}
```
move this assignment to after the loop, as it only needs to happen once when we're done filling `newImages`